src/main/scala/firrtl/analyses/CircuitGraph.scala

```diff
@@ -48,10 +49,10 @@ class CircuitGraph private[analyses] (connectionGraph: ConnectionGraph) {
   private val irLookup = connectionGraph.irLookup
 
   // Module/Instance Hierarchy information
-  private lazy val instanceGraph = new InstanceGraph(circuit)
+  private lazy val instanceGraph = new InstanceKeyGraph(circuit)
```

Comment:
Can we actually make the constructor for `InstanceKeyGraph` private and instead use a factory method on the companion object? Makes future refactoring easier.

Sorry, this actually should have been a comment on https://github.com/freechipsproject/firrtl/pull/1732

Reply:
Sure

Reply:
Done